src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/SystemIPv4InterfaceProperties.cs

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+
+using System.Runtime.InteropServices;
+
+namespace System.Net.NetworkInformation
+{
+    internal class SystemIPv4InterfaceProperties : IPv4InterfaceProperties
+    {
+        // These are only valid for ipv4 interfaces.
+        private bool _haveWins = false;
+        private bool _dhcpEnabled = false;
+        private bool _routingEnabled = false;
+        private bool _autoConfigEnabled = false;
+        private bool _autoConfigActive = false;
+        private uint _index = 0;
+        private uint _mtu = 0;
```

Comment:
This is probably true for most/all of the "Properties/Statistics" classes, they're just grab bags of data that doesn't update unless you get a new object.